clients/unshaded/src/main/java/tachyon/client/block/TachyonBlockStore.java

```diff
@@ -105,8 +109,14 @@ public BufferedBlockInStream getInStream(long blockId) throws IOException {
         NetAddress workerNetAddress = location.getWorkerAddress();
         if (workerNetAddress.getHost().equals(localHostName)) {
           // There is a local worker and the block is local.
-          return new LocalBlockInStream(blockId, blockInfo.getLength(),
-              new InetSocketAddress(workerNetAddress.getHost(), workerNetAddress.getDataPort()));
+          try {
+            return new LocalBlockInStream(blockId, blockInfo.getLength(),
+                new InetSocketAddress(workerNetAddress.getHost(), workerNetAddress.getDataPort()));
+          } catch (IOException e) {
+            LOG.warn("Failed to open local stream for block " + blockId + ". " + e.getMessage());
```

Comment:
we don't do this is other places, but it could be nicer to do `LOG.warn("Failed to open local stream for block %s: %s", blockId, e.getMessage())` for +readability and +performance